rl4co/envs/routing/cvrp/env.py

```diff
@@ -257,7 +256,7 @@ def replace_selected_actions(
     def local_search(td: TensorDict, actions: torch.Tensor, **kwargs) -> torch.Tensor:
         assert (
             local_search is not None
-        ), "Cannot import local_search module. Check if `pyvrp` is installed."
+        ), "Cannot import local_search module. Check if HGS is built."
```

Comment:
Perhaps some instructions required here to tell users on how to build HGS, e.g. "Please check ... for more information."